docs/use-github-in-bcgov/bc-government-organizations-in-github.md

```diff
@@ -1,120 +1,96 @@
 # B.C. Government organizations in GitHub
 
-Many teams building software for the B.C. government use [GitHub](https://github.com) to host their code. Using GitHub you can:
+The [Digital Principles for the Government of B.C.](https://www2.gov.bc.ca/gov/content/governments/policies-for-government/core-policy/policies/im-it-management#12.1.1.5) urge product teams to work in the open. GitHub is the leading platform for open-source projects. It allows our organization to collaborate with the open-source community to build software, support innovation and save time and money. Using GitHub, teams can:
 
+* Collaborate with the open-source community
 * Share and control code versions
 * Use tools for team and project management
-* Track issues
-* Collaborate with the open-source community
 * Integrate automation tools
+* Track issues
 
-The main organization the B.C. government owns in GitHub is [bcgov](https://github.com/bcgov) where we store all open-source code developed by B.C. government teams. The `bcgov` organization includes over 1000 repositories maintained by the B.C. government developer community.
-
-The Developer Experience Team manages the B.C. government's GitHub environments.
-
-## Working in the open
+B.C. Government's main organization is [bcgov](https://github.com/bcgov), where we store all open-source code developed by government teams. The `bcgov` organization includes over 1000 repositories. We also have the `bcgov-c`organization, where developers can either permanently or temporarily store confidential code. 
 
-The [Digital Principles for the Government of B.C.](https://www2.gov.bc.ca/gov/content/governments/policies-for-government/core-policy/policies/im-it-management#12.1.1.5) urge product teams to work in the open as outlined in principle five. GitHub is the leading platform for open-source projects. It allows the Province to collaborate with the open-source community to build software, support innovation and save time and money.
+On behalf of the B.C. Government, the Developer Experience Team manages government GitHub environments.
 
 ## Organizations in GitHub
 
-The Province owns several GitHub organizations, which are described below.
+We use GitHub Enterprise Cloud, and it hosts the `bcgov` and `bcgov-c` GitHub organizations: 
```

Comment:
I wonder if it's worth having some change management language here so folks know this is new/different than what they might have known in the past? a little clue not to skim

Reply:
@jenreiher-bcgov if we go with the version or similar in the comment, I'll remove the last sentence: 

Recently, we enabled Single Sign-On (SSO) for `bcgov` and on **August 21, 2024**, we'll enable SSO for `bcgov-c`. The transition allowed us to understand user needs better, and streamline services.  

Now, we use GitHub Enterprise Cloud, and it hosts the `bcgov` and `bcgov-c` GitHub organizations: 

```mermaid
  graph TD;
A[GitHub Enterprise Cloud] --> B[bcgov]
A --> C[bcgov-c]
```